Intersect.Client/Localization/Strings.cs

```diff
@@ -1551,6 +1551,12 @@ public struct Settings
 
             [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
             public static LocalizedString GameSettingsTab = @"Game";
+            
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+            public static LocalizedString GraphicalUserInterfaceSettings = @"Interface";
```

Comment:
In the comments you have `// Game Settings - Game User Interface (GUI)` but then you keep calling it GraphicalUserInterface....

Can you just remove "GU"/"GraphicalUser" and just call it all "Interface" instead?